docs/source/configuration/authn-jwt.mdx

```diff
@@ -139,7 +139,7 @@ The default value is `Authorization`.
 
 The string that will always precede the JWT in the header value corresponding to [`header_name`](#header_name). This value must not include whitespace.
 
-The default value is `Bearer`.
+The default value is `Bearer`. If the router encounters authorization headers with a different prefix, it will ignore them.
```

Comment:
The current default does the opposite, no? It will reject the request as far as I can tell.

Reply:
Hmm. I would guess they may use it as an expected behavior to fail if it isn't formatted correctly, including overall header structure. I can reach out to other SAs to get a gut check.